packages/studio-base/src/panels/RawMessages/index.tsx

```diff
@@ -623,6 +614,74 @@ function RawMessages(props: Props) {
     valueRenderer,
   ]);
 
+  const actionHandler = useCallback((action: SettingsTreeAction) => {
+    if (action.action === "update") {
+      if (action.payload.path[0] === "general") {
+        if (action.payload.path[1] === "fontSize") {
+          setCustomFontSize(
+            action.payload.value != undefined ? (action.payload.value as number) : undefined,
+          );
+        } else if (action.payload.path[1] === "fontWeight") {
+          setCustomFontWeight(
+            action.payload.value != undefined ? (action.payload.value as number) : undefined,
+          );
+        }
+      }
+    }
+  }, []);
+
+  useEffect(() => {
+    updatePanelSettingsTree({
+      actionHandler,
+      nodes: {
+        general: {
+          label: "General",
+          fields: {
+            fontSize: {
+              label: "Font Size",
+              input: "select",
+              options: [
+                ...Constants.FONT_SIZE_OPTIONS.map((value) => ({
+                  label: `${value} px` + (value === theme.typography.fontSize ? ` (Default)` : ``),
+                  value,
+                })),
+              ],
+              value: customFontSize,
+            },
+            fontWeight: {
+              label: "Font Weight",
+              input: "select",
+              options: [
+                {
+                  label: "Regular",
+                  value: theme.typography.fontWeightRegular as number,
+                },
+                {
+                  label: "Medium",
+                  value: theme.typography.fontWeightMedium as number,
+                },
+                {
+                  label: "Bold",
+                  value: theme.typography.fontWeightBold as number,
+                },
+              ],
+              value: customFontWeight,
```

Comment:
should we have default placeholder values for these?

Reply:
Right now it shows this by default because this `value` field is `undefined`: 
<img width="324" alt="image" src="https://github.com/foxglove/studio/assets/10187776/a706629b-d774-44ed-9612-a3e18e018ffc">
By doing something like this:
```suggestion
              value: customFontWeight ?? (theme.typography.fontWeightRegular as number),
```
Doing this we can get rid of the red border and show what we use by default when `customFontWeight` (and `customFontSize`) are undefined.